designsafe/apps/api/projects/managers/publication.py

```diff
@@ -166,13 +166,36 @@ def _project_required_xml(project, authors, created, doi=None):
         identifier.text = SHOULDER.replace('doi:', '')
     creators = ET.SubElement(resource, 'creators')
 
-    for author in authors:
-        if not author.get('lname') and not author.get('fname'):
-            logger.error('Author in wrong format: %s', author)
-            continue
-        creator = ET.SubElement(creators, 'creator')
-        creator_name = ET.SubElement(creator, 'creatorName')
-        creator_name.text = '{}, {}'.format(author['lname'], author['fname'])
+    if project.value.projectType == 'other':
```

Comment:
Having difficulty tracking down the data type of `project` without actually publishing a project. I believe it is formatted as a request, but I'm not sure. The ordering of authors on the XML is working, but I'm not sure if `project.value.projectType` is going to work in this case.